### DISCUSSION TITLE
"draw groundtruths" and "draw detections" does not draw!

### DISCUSSION THREAD

**Discussion by h4ppy0wl** [rafaelpadilla/review_object_detection_metrics#147] [Q&A]
Hi,

Thanks for sharing this valuable repo and research results.
I created the COCO style ground truth and detections json files (1 json file for gt annotations in 89 images and 1 json file for detections in the same 89 images) based on the examples in the repo. then installed and followed the instructions to use the project.
While I get detection metrics, **I can not view ground truth and detection boxes by selecting the checkmarks**. 
Each image can have up to 25 gt and/or detections. All image sizes are 1024x1024. 

**Would you please guide me in troubleshooting the problem?** 

![odm0](https://github.com/user-attachments/assets/968ca519-e202-49fb-b121-befae182043c)
![odm1](https://github.com/user-attachments/assets/5f59a91e-835e-4aa9-aba8-ea4794723443)
![odm2](https://github.com/user-attachments/assets/d0f41403-6ecd-4728-bfd7-eda12370c582)
![dt](https://github.com/user-attachments/assets/41af0f83-e80d-4317-b962-785eadb28fd5)
![gt](https://github.com/user-attachments/assets/862aeba9-6638-44de-a0a2-0465fb61a59f)

**Comment by rafaelpadilla** [comment]
That seems strange. Is it possible to send me the json files and one of your sample images?

**Reply by h4ppy0wl** [reply]
I am truly sorry for this long delay in responding!!
It was a mistake by me in my detection output. soved.
Thanks
